### PR TITLE
[GCI-267] React UI Library - Dropdown Component

### PR DESCRIPTION
**Pull request by TomaszDomagala**
#### Issue: https://issues.openmrs.org/browse/GCI-267
Props:
- value: first option
- options, array of options (objects with 'value' and 'text' fields)
- isDisabled, boolean
- onChangeCallback, function with one value param
- width
- size (number of options)

### CODE REVIEW

File: src/components/DropDown/index.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import PropTypes from 'prop-types';
+import './dropdown.css'
+
+class DropDown extends Component {
+
+    render() {
+        let options = this.props.options.map(function (option) {
+            return <option>{option}</option>
+        });
+        if (this.props.text !== "" && this.props.text !== null)
```
Comment (merovingienne) [inline src/components/DropDown/index.jsx]: Can be done with a single `if (this.props.text) {...}`.

**Comment by TomaszDomagala** [comment]
@merovingienne I've fixed issues here I think

**Comment by merovingienne** [comment]
@tomekdo3 Great, I've given feedback on GCI.

**Comment by TomaszDomagala** [comment]
@merovingienne what do you think about that? 😊

**Pull request closed without merging**